driver-core/src/main/com/mongodb/client/model/Accumulators.java

```diff
@@ -321,6 +492,24 @@ private static <TExpression> BsonField accumulatorOperator(final String name, fi
         return new BsonField(fieldName, new SimpleExpression<TExpression>(name, expression));
     }
 
+    private static <InExpression, NExpression> BsonField pickNAccumulator(
+            final String fieldName, final String accumulatorName, final InExpression inExpression, final NExpression nExpression) {
+        return new BsonField(fieldName, new Document(accumulatorName, new Document("input", inExpression).append("n", nExpression)));
+    }
+
+    private static <OutExpression, NExpression> BsonField sortingPickAccumulator(
```

Comment:
`NExpression` is unused and can be removed.

Reply:
Thanks, done.